scripts/common.sh

```diff
@@ -61,7 +61,7 @@ build_kernel()
 				run_cmd git checkout current/${BRANCH}
 				COMMIT=$(git log --format="%h" -1 HEAD)
 
-				run_cmd "cp /boot/config-$(uname -r) .config"
+				run_cmd "cp /boot/config-$(uname -r) .config || cat /proc/config.gz | gunzip > .config"
```

Comment:
Not all systems will have `/proc/config.gz` either (you need CONFIG_IKCONFIG_PROC=y). Also, you can use `zcat /proc/config.gz` instead of piping through `gunzip`.

Reply:
> Not all systems will have /proc/config.gz either 

yeah that's true. this is just for one workaround that works in my environment..
changed to use `zcat`.